protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -190,7 +190,10 @@ func buildEtcdEnvironmentOptions(c *EtcdCluster) []v1.EnvVar {
 		{Name: "ETCD_ADVERTISE_CLIENT_URLS", Value: fmt.Sprintf("%s://%s:%d", scheme, c.Me.InternalName, c.ClientPort)},
 		{Name: "ETCD_INITIAL_ADVERTISE_PEER_URLS", Value: fmt.Sprintf("%s://%s:%d", scheme, c.Me.InternalName, c.PeerPort)},
 		{Name: "ETCD_INITIAL_CLUSTER_STATE", Value: "new"},
-		{Name: "ETCD_INITIAL_CLUSTER_TOKEN", Value: c.ClusterToken}}...)
+		{Name: "ETCD_INITIAL_CLUSTER_TOKEN", Value: c.ClusterToken},
+		{Name: "ETCD_LISTEN_METRICS_URLS", Value: fmt.Sprintf("%s://__address__:%d", scheme, c.MetricsPort)},
+		{Name: "ETCD_METRICS", Value: "basic"},
```

Comment:
I'm adding this despite this is default value for better visibility. Should be configurable (MetricsPort probably too).

Reply:
@michalschott it looks to me like these would belong in the EtcdClusterSpec struct.